content/news/2023-03-04-bevy-0.10/index.md

```diff
@@ -1291,6 +1291,24 @@ fn run_or_attack(
 [`AnimationPlayer`]: https://docs.rs/bevy/0.10.0/bevy/animation/struct.AnimationPlayer.html
 [`play_with_transition`]: https://docs.rs/bevy/0.10/bevy/animation/struct.AnimationPlayer.html#method.play_with_transition
 
+### AcessKit integration into `bevy_ui`
+
+<div class="release-feature-authors">authors: @ndarilek</div>
+
+Games are for everyone: and the way they're built should reflect that.
+Acessible games are rare, and proper support is often an afterthought, both at an engine and a game level.
+By building our UI solution with acessibility in mind, we hope to fix that.
```

Comment:
s/acessibility/accessibility/ Sorry, not sure how to make that a suggested change. Thanks for writing this up!